### PR TITLE
2.7.0

### PR DESCRIPTION
**Pull request by ebraminio**
Expected to be published in some days (as I expect some new fuzzing issues to pop out and am softly waiting for #2553 also somehow), uploaded however to get feedback on the release note, is the release note too verbose or has grammar/spell mistakes, or not happy to with things it suggests? Feel free to give me your feedback :)

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,36 @@
+Overview of changes leading to 2.7.0
+Thursday, July 9, 2020
+====================================
+- Use an always round up implementation for round, some minor fluctuations is
```
Comment (alerque) [inline NEWS:4]:
```suggestion
- Use an an implementation for round that always rounds up, some minor fluctuations is
```

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+  expected on var font specially when hb-ot callback is used.
+- Fix an AAT's `kerx` issue on broken rendering of Devanagari Sangam MN.
+- Remove AAT's `lcar` table support from _get_ligature_carets API, not even much
+  use on macOS installed fonts (only two files).  GDEF support is the recommended
+  one and expected to work properly after issues fixed two release ago.
+- Minor memory fixes to handle OOM better specially in hb-ft.
+- Minor .so files versioning scheme change and remove stable/unstable scheme
+  differences was never used in action (always defaulting to stable scheme).
```
Comment (alerque) [inline NEWS:12]:
```suggestion
  differences, was never used in practice (always default to stable scheme).
```

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+- *Migration to Meson build system for building the package*
+  As ArchLinux, AlpineLinux and msys2 have migrated to the meson port, we have
+  reached to a confidence that the meson port is probably usable for packaging
```
Comment (alerque) [inline NEWS:15]:
```suggestion
  become confident that the meson port is probably usable for packaging
```

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+  the library but of course feedback still is welcomed and some minor issues
```
Comment (alerque) [inline NEWS:16]:
```suggestion
  the library -- but of course feedback is still welcomed and some minor issues
```

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,36 @@
...
+  git+https://github.com/harfbuzz/harfbuzz#commit=$_commit (used by ArchLinux)
+  which is much more safe and reliable in terms of knowing which
+  exact changes has led to the release.
+  Are you just integrating the library into your software and don't have the cycles
+  to migrate to meson for now?  There is also amalgam/unity support,
+  src/harfbuzz.cc, a single source file that includes the whole library, provided
+  along the source but that needs a minor study of build configs meson or autotools
+  do on flags and definitions they pass to the compiler thus we recommend use of
+  the build system for regular cases and specially don't recommend the use amalgam
+  for *nix packages in any case.
```
Comment (alerque) [inline NEWS:31]:
```suggestion
  If you are just integrating the library into your software and don't have the cycles
  to migrate to meson for now, there is also amalgam/unity support, src/harfbuzz.cc.
  This is a single source file that includes the whole library, provided along the
  source. However using it requires a minor study of what the build configs (meson or
  autotools) do with flags and definitions that they pass to the compiler -- thus we
  recommend use of the regular build system for most cases and specially recommend
  against the use of the amalgam for *nix packages.
```

Comment (ebraminio) [reply NEWS]: Done

File: NEWS (hunk 4 above)
Comment (ebraminio) [reply NEWS]: Done

File: NEWS (hunk 3 above)
Comment (ebraminio) [reply NEWS]: Done

File: NEWS (hunk 2 above)
Comment (ebraminio) [reply NEWS]: Done

File: NEWS (hunk 1 above)
Comment (ebraminio) [reply NEWS]: Done

**Comment by ebraminio** [comment]
@alerque, niiiice, thank you very much :)

File: NEWS [hunk 6]
```diff
@@ -1,3 +1,36 @@
+Overview of changes leading to 2.7.0
+Thursday, July 9, 2020
+====================================
+- Use an an implementation for round that always rounds up, some minor
+  fluctuations is expected on var font specially when hb-ot callback is used.
```
Comment (alerque) [inline NEWS:5]:
```suggestion
- Use an implementation for round that always rounds up, some minor
  fluctuations are expected on var font specially when hb-ot callback is used.
```

File: NEWS [hunk 7]
```diff
@@ -1,3 +1,36 @@
...
+- Fix an AAT's `kerx` issue on broken rendering of Devanagari Sangam MN.
+- Remove AAT's `lcar` table support from _get_ligature_carets API, not even much
+  use on macOS installed fonts (only two files).  GDEF support is the recommended
+  one and expected to work properly after issues fixed two release ago.
```
Comment (alerque) [inline NEWS:9]:
```suggestion
  one and expected to work properly after issues fixed two releases ago.
```

File: NEWS (hunk 6 above)
Comment (ebraminio) [reply NEWS]: Done

File: NEWS (hunk 7 above)
Comment (ebraminio) [reply NEWS]: Done

**Comment by ebraminio** [comment]
> More copy edits I missed (and one I buggered up)...

Oops, thank you very much! :)

**Comment by khaledhosny** [comment]
Can we merge #2553 before the release?

**Comment by khaledhosny** [comment]
(or at least revert the bad commit until the PR is reviewed, the issues it is causing seem to be more serious than the issue it fixed).

File: NEWS [hunk 8]
```diff
@@ -1,3 +1,36 @@
...
+  git+https://github.com/harfbuzz/harfbuzz#commit=$_commit (used by Arch Linux)
+  which is much more safe and reliable in terms of knowing which
+  exact changes has led to the release.
+  If you are just integrating the library into your software and don't have the
+  cycles to migrate to meson for now, there is also amalgam/unity support,
+  src/harfbuzz.cc. This is a single source file that includes the whole library,
+  provided along the source. However using it requires a minor study of what the
+  build configs (meson or autotools) do with flags and definitions that they pass
+  to the compiler -- thus we recommend use of the regular build system for most
+  cases and specially recommend against the use of the amalgam for *nix packages.
```
Comment (khaledhosny) [inline NEWS:31]: I don’t kmow, but this entry feels to much for NEWS files. It should be shorter, a line or two with a link to documentation (wiki) page with more details. Something like:
```
- Migration to meson is progressing, and now is good time to try it and report any issues.
- Source URL changed, please use the new URL: XXX
  For more information, see XXX
```

Comment (alerque) [reply NEWS:31]: I kind of agree. How and when to use (and not use) the alternative library build method is probably better placed in docs somewhere with just a mention of the priority shift in NEWS.

Comment (ebraminio) [reply NEWS]: Great idea! Moved most of it to https://github.com/harfbuzz/harfbuzz/wiki/Notes-on-migration-to-meson feel free to edit that (or even rename the page)

**Comment by ebraminio** [comment]
> Can we merge #2553 before the release?

As mentioned on OP :) And am trusting on you on decision you make there :)

**Pull request merged**